frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -123,16 +123,7 @@ export const ClusterServiceVersionList: React.SFC<ClusterServiceVersionListProps
   return <Table {...props} aria-label="Installed Operators" Header={ClusterServiceVersionTableHeader} Row={ClusterServiceVersionTableRow} EmptyMsg={EmptyMsg} virtualize />;
 };
 
-const stateToProps = ({k8s, FLAGS}) => ({
-  loading: FLAGS.get(featureFlags.OPENSHIFT) === undefined || !k8s.getIn([FLAGS.get(featureFlags.OPENSHIFT) ? 'projects' : 'namespaces', 'loaded']),
-});
-
-export const ClusterServiceVersionsPage = connect(stateToProps)((props: ClusterServiceVersionsPageProps) => {
-  // Wait for OpenShift feature detection to prevent flash of "disabled" UI
-  if (props.loading) {
-    return <LoadingBox />;
-  }
-
+export const ClusterServiceVersionsPage = (props: ClusterServiceVersionsPageProps) => {
```

Comment:
Going to need to remove `.WrappedComponent` from [the unit test as well](https://github.com/openshift/console/blob/master/frontend/__tests__/components/operator-lifecycle-manager/clusterserviceversion.spec.tsx#L127).